@xen-orchestra/lite/src/components/UsageBar.vue

```diff
@@ -37,7 +39,7 @@ interface Data {
 }
 
 interface Props {
-  data?: Data[];
+  data?: Data[] | null;
```

Comment:
I'm not fond of relying on data type to detect errors, it's really confusing.

Beyond that, I don't think it's up to this component do handle the error display.

Instead of doing something like this:
```vue
<UsageBar
  :data="hasError ? null : isReady ? data.result : undefined"
  :nItems="N_ITEMS"
>
```

It would be preferable to do:
```vue
<NoDataError v-if="hasError" />
<UsageBar
  v-else
  :data="isReady ? data.result : undefined"
  :nItems="N_ITEMS"
>
```
Which is less confusing, IMO.